actions/setup-github-token/action.yml

```diff
@@ -27,13 +32,24 @@ outputs:
 runs:
   using: composite
   steps:
+    - name: Check the role session name lengths and truncate if needed
+      shell: bash
+      id: role-session-name
+      env:
+        ROLE_SESSION_NAME: ${{ inputs.role-session-name }}
+      run: |
+        if [[ ${#ROLE_SESSION_NAME} -gt 64 ]]; then
+          echo "role-session-name=$(echo "$ROLE_SESSION_NAME" | cut -c1-64)" >> "$GITHUB_OUTPUT"
+        fi
+
     - name: Assume role capable of getting token from gati
       uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
       with:
-        role-to-assume: ${{ inputs.aws-role-arn }}
-        role-duration-seconds: ${{ inputs.aws-role-duration-seconds }}
         aws-region: ${{ inputs.aws-region }}
         mask-aws-account-id: true
+        role-duration-seconds: ${{ inputs.aws-role-duration-seconds }}
+        role-session-name: ${{ inputs.role-session-name }}
```

Comment:
Won't this still use the potentially too long `${{ inputs.role-session-name }}`?

Reply:
We truncate it here to a maximum of 64 characters, regardless of whether the value is created using the default value or passed from another GH action:

https://github.com/smartcontractkit/.github/pull/418/files#diff-8b60da546acdf6951c641aa7ec7432c6923ff15b0d141f2db9ef309d3bb03be6R41

Reply:
That's not modifying the `${{ inputs.role-session-name }}` though is it?  You're piping that truncated value to `$GITHUB_OUTPUT`.

Reply:
That's right. I'll put a fix, thank you.

Reply:
ahh yeah sorry - good catch!